src/System.Text.Json/tests/NewtonTests/EnumConverterTests.cs

```diff
@@ -0,0 +1,532 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public class EnumConverterTests
+    {
+        public class EnumClass
+        {
+            public StoreColor StoreColor { get; set; }
+            public StoreColor NullableStoreColor1 { get; set; }
+            public StoreColor? NullableStoreColor2 { get; set; }
+        }
+
+/*        public class EnumContainer<T>
```

Comment:
Same here/elsewhere - what's the plan for the commented out code here?

Reply:
The feature is not supported, but coming in the future or good to have feature, we talked to keep these tests as they are good to have. But most likely this feature not coming in near future, so maybe remove these tests

Reply:
I don't think its worth checking in yet. Could we keep track of it separately (either locally, as part of the commit history, or copied into a [gist](http://gist.github.com))?

Reply:
OK